### ISSUE TITLE
Error when the mask appears ( Mirage + LethalIntelligenceExperimental )

### ISSUE THREAD

**Issue by emYons** [qwbarch/mirage#113]
Checked on a clean list of mods, only Mirage + LethalIntelligenceExperimental and their dependencies.
When the mask appears, an error in the console constantly appears, and the mob's behavior breaks and it goes into the wall.

```
[Warning: Unity Log] Invalid Layer Index '-1'
[Error  : Unity Log] NullReferenceException: Object reference not set to an instance of an object
Stack trace:
LethalIntelligence.Patches.MaskedAIRevamp.CheckForEntrancesNearby () (at ./Patches/MaskedAIRevamp.cs:445)
LethalIntelligence.Patches.MaskedAIRevamp.FixedUpdate () (at ./Patches/MaskedAIRevamp.cs:1488)
```

![picture_2024-12-09_235154028](https://github.com/user-attachments/assets/bf124bd2-4fcb-4757-a722-5c574686453f)

**Comment by qwbarch** [comment]
LethalIntelligence isn't compatible with the latest Mirage update. Virus is currently working on a fix, sorry about that

**Comment by qwbarch** [comment]
FYI for now, you'll either want to turn off Mirage compatibliity in LI's config, or stick to the previous version of Mirage if that worked fine for you

**Comment by qwbarch** [comment]
Mirage + LethalIntelligence compatibility is fixed in [LethalIntelligence v0.3.3](https://thunderstore.io/c/lethal-company/p/VirusTLNR/LethalIntelligenceExperimental/)
